src/actions/index.js

```diff
@@ -30,6 +35,15 @@ export const deleteTodo = (id) => {
     id,
   };
 };
+
+export const completeTodo = (id, date) => {
+  return {
+    type: COMPLETE_TODO,
+    id,
+    isTaskComplete: true,
+    completeTime: getDays(date.getTime(), new Date().getTime()),
+  };
```

Comment:
see https://redux.js.org/tutorials/essentials/part-4-using-data#preparing-action-payloads

```suggestion
  return {
    type: COMPLETE_TODO,
    payload: {
        id,
        isTaskComplete: true,
        completeTime: getDays(date.getTime(), new Date().getTime()),
    }
  };
```